_config.yml

```diff
@@ -62,3 +63,14 @@ author:
 
 # URLs
 img_url: '/img/'
+
+# Site settings
+baseurl: ""  # leave empty if serving from root
+url: ""      # leave empty for local development
+
+# Asset settings
+defaults:
+  - scope:
+      path: "assets/images"
+    values:
+      image: true
```

Comment:
Image handling is all setup. I'll comment inline below with fixes.

I think everything in `_config.yml` here can be reverted.

----

RE: `baseurl:` and `url:` 

Were you having issues previewing the site locally?